README.md

```diff
@@ -0,0 +1,76 @@
+# AI Palatal Rugoscopy
+
+Instance segmentation of palatal rugae patterns using Deep Learning.
+
+
+## Development environment
+By default, we use a [Docker](https://www.docker.com/) container as development environment. \
+Before building the image, please ensure you have the following resources available:
+
+- GPU with an updated driver version
+- nvidia-container-tookit (allows running GPU-accelerated containers)
+
+The script [build-container.sh](docker/build-container.sh) automatically builds the image and creates the container. To execute this script, run the following command in a bash terminal:
+
+```shell
+cd docker
+bash build_container.sh --data-path <DATA_PATH>
+```
+Where:
+```
+DATA_PATH is the path to a folder containing the dataset (images).
+```
+
+## Dataset Preprocessing
+There are a few scripts available to preprocess data. They can be found in the [standalone](src/standalone/) directory.
+
+- `dataset_split.py`: Splits a dataset into `train`, `test` and `validation` sets. Usage: 
+```shell
+python dataset_split.py --annotations-path <ANNOTATION_FILE> --output-path <OUTPUT_DIRECTORY> --split <TRAIN_SIZE> <VALIDATION_SIZE> [<TEST_SIZE>]
+```
+Where
+```
+ANNOTATION_FILE: Path to annotation file.
+OUTPUT_DIRECTORY: Path to the output file.
+TRAIN_SIZE: Percentage of samples to be allocated in the train subset.
+VALIDATION_SIZE: Percentage of samples to be allocated in the validation subset.
+TEST_SIZE (Optional): Percentage of samples to be allocated in the test subset.
+```
+
+- `dataset_patches_generation.py`: Break images into patches. Usage:
+
```

Comment:
You're not very consistent with your line breaks, it appears? 😆 
```suggestion
```

Reply:
Indeed :laughing:. It is all about avoiding ugly blocks of code/text. We can establish a pattern if you wish.

Reply:
Most of what I'm suggesting here is that both titles and code blocks are separated by empty lines (above and below) from the rest of the text.

Reply:
The only exception could be (but I honestly prefer it to be the same) code blocks inside bullet points.